app/assets/javascripts/components/overview/details.jsx

```diff
@@ -314,62 +315,97 @@ const Details = createReactClass({
+                showTime={this.props.course.use_start_and_end_times}
+                required={true}
+              />
+              <DatePicker
+                onChange={this.updateCourseDates}
+                value={this.props.course.end}
+                value_key="end"
+                editable={this.props.editable}
+                validation={CourseDateUtils.isDateValid}
+                label={CourseUtils.i18n('end', this.props.course.string_prefix)}
+                date_props={dateProps.end}
+                enabled={Boolean(this.props.course.start)}
+                showTime={this.props.course.use_start_and_end_times}
+                required={true}
+              />
+            </form>
+            {campaigns}
+          </div>
         </div>
-        <div className="module__data extra-line-height">
-          <Instructors {...this.props} />
-          {online}
-          {campus}
-          {staff}
-          <div><p className="red">{this.state.error_message}</p></div>
-          {school}
-          {title}
-          {term}
-          <form>
-            {passcode}
-            {expectedStudents}
-            <DatePicker
-              onChange={this.updateCourseDates}
-              value={this.props.course.start}
-              value_key="start"
-              validation={CourseDateUtils.isDateValid}
-              editable={this.props.editable}
-              label={CourseUtils.i18n('start', this.props.course.string_prefix)}
-              showTime={this.props.course.use_start_and_end_times}
-              required={true}
-            />
-            <DatePicker
-              onChange={this.updateCourseDates}
-              value={this.props.course.end}
-              value_key="end"
-              editable={this.props.editable}
-              validation={CourseDateUtils.isDateValid}
-              label={CourseUtils.i18n('end', this.props.course.string_prefix)}
-              date_props={dateProps.end}
-              enabled={Boolean(this.props.course.start)}
-              showTime={this.props.course.use_start_and_end_times}
-              required={true}
-            />
-            {timelineStart}
-            {timelineEnd}
-          </form>
-          {campaigns}
-          {subject}
-          {courseLevelSelector}
-          {tags}
-          {courseTypeSelector}
-          {submittedSelector}
-          {privacySelector}
-          {timelineToggle}
-          {withdrawnSelector}
-          {projectSelector}
-          {languageSelector}
+      );
+    }
+
+    return (
+      <Modal>
+        <div className="module course-details modal-course-details">
+          <div className="section-header">
+            <h3>{I18n.t('application.details')}</h3>
+            {this.props.controls()}
+          </div>
+          <div className="module__data extra-line-height">
+            <Instructors {...this.props} />
+            <div className="details-form">
+              <div className="group-left">
```

Comment:
Hmm... it's still a lot of duplication, I guess because of these extra divs. Would it work to keep the same arrangement in both modes, and just switch the classnames in edit mode?